### PR TITLE
feat: add slogan parser for "at"

### PR DESCRIPTION
**Pull request by tiger2005**
呵呵，啊哈哈哈哈哈哈…大便来喽！

### CODE REVIEW

File: src/data/interfaces/types.ts [hunk 1]
```diff
@@ -39,3 +39,9 @@ export interface ColorTable {
   blue: string,
   gold: string
 }
+
+export interface SearchResult {
+  1: string,
+  index: number,
+  endIdx: number
+}
```
Comment (Nikaidou-Shinku) [inline src/data/interfaces/types.ts]: new line at end pls.

File: src/data/LuoguAPI.ts [hunk 2]
```diff
@@ -23,3 +23,7 @@ export const getFansPage = (uid: number) => {
 export const getPracticePage = (uid: number) => {
   return `https://www.luogu.com.cn/user/${uid}#practice`;
 };
+
+export const getSearch = (name: string) => {
+  return `https://www.luogu.com.cn/api/user/search?keyword=${name}`;
+}
```
Comment (Nikaidou-Shinku) [inline src/data/LuoguAPI.ts]: new line at end pls.

File: src/components/index.ts [hunk 3]
```diff
@@ -4,3 +4,4 @@ export { Card } from "./Card";
 export { InfoCard } from "./InfoCard";
 export { FailedCard } from "./FailedCard";
 export { LoadingCard } from "./LoadingCard";
+export { loadAddress } from "./AddressLoader";
```
Comment (Nikaidou-Shinku) [inline src/components/index.ts]: new line at end pls.

**Review by Nikaidou-Shinku — CHANGES_REQUESTED** [review]
摆烂了，也没什么不好的，我就做下基本的 code style check 算了，逻辑懒得深究了……

File: src/data/LuoguAPI.ts (hunk 2 above)
Comment (tiger2005) [reply src/data/LuoguAPI.ts]: Done.

File: src/components/index.ts (hunk 3 above)
Comment (tiger2005) [reply src/components/index.ts]: Done.

File: src/data/interfaces/types.ts (hunk 1 above)
Comment (tiger2005) [reply src/data/interfaces/types.ts]: Done.

File: src/components/AddressLoader.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                if (-- unfAjax === 0)
+                    doCallback();
+            },
+            error: () => {
+                if (-- unfAjax === 0)
+                    doCallback();
+            }
+        });
+    }
+};
```
Comment (Nikaidou-Shinku) [inline src/components/AddressLoader.tsx]: new line at end pls.

**Review by Nikaidou-Shinku — CHANGES_REQUESTED** [review]

File: src/components/AddressLoader.tsx (hunk 4 above)
Comment (tiger2005) [reply src/components/AddressLoader.tsx]: ……ok

**Review by Nikaidou-Shinku — APPROVED** [review]

**Pull request merged**